ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml

```diff
@@ -59,6 +59,9 @@ parameters:
   required: false
 - name: MULTIARCH_RELEASE_IMAGE_LATEST
   required: true
+- name: USE_LEASE_CLIENT
```

Comment:
is this used anywhere?

Reply:
I added this is a previous run and I thought it fixed something. I'm no longer convinced it does anything, but I am loath to wait another cycle to try to remove this. Perhaps you can add it as a fixup patch to your 4.6 job? (That way we can resolve the current crisis ASAP)

Reply:
Actually, I'll file a follow up PR to fix this and the bridge name bug that causes the s390x 4.3 test to fail.

Reply:
yah, i was super hesitant to even ask that question for the same reason. :D sounds good to me!